doc/apiv3-documentation.apib

```diff
@@ -1958,9 +1964,15 @@ This endpoint lists the types that are *available* in a given project.
 | name        | User's full name, formatting depends on instance settings | String   |             | READ                 |
 | mail        | User's email                                              | String   |             | READ                 |
 | avatar      | URL to user's avatar                                      | String   |             | READ                 |
+| status      | The current activation status of the user (see below)     | String   |             | READ                 |
 | createdAt   | Time of creation                                          | DateTime |             | READ                 |
 | updatedAt   | Time of the most recent change to the user                | DateTime |             | READ                 |
 
+The `status` of a user can be one of:
+
+* `active` - the user can act normally
+* `registered` - the user just registered to the instance, activation by an administrator is required
+* `locked` - the user was locked and can't login
```

Comment:
Just some questions of wording here:
* what do you mean by "can act normally"? I wouldn't want to use the word "normally" to describe my behaviour at the best of times.
 * Proposal: "The user can log in with the account" 
* ```activation by an administrator is required```. He could also be in the first phase of registering himself. He would then have to activate his account by following the link in the e-mail.
 * Proposal: "The user can not log in yet but will be able to once the registration is completed."
* ```the user was locked and can't login``` the account is still locked so "was" seems wrong to me.